index.html

```diff
@@ -1261,6 +1273,19 @@ <h3>Algorithm</h3>
                   error has been detected and processing is aborted.</li>
               </ol>
             </li>
+            <li class="changed">If <var>context</var> has an <code>@direction</code> <a>entry</a>:
+              <ol>
+                <li>Initialize <var>value</var> to the value associated with the
+                  <code>@direction</code> <a>entry</a>.</li>
+                <li>If <var>value</var> is <code>null</code>, remove
+                  any <a>base direction</a> from <var>result</var>.</li>
+                <li>Otherwise, if <var>value</var> is <a>string</a>, the
+                  <a>base direction</a> of <var>result</var> is set to
+                  <var>value</var>. If it is not `null`, `"ltr"`, or `"rtl"`, an
+                  <a data-link-for="JsonLdErrorCode">invalid base direction</a>
+                  error has been detected and processing is aborted.</li>
```

Comment:
Do we need to consider whether or not `@direction` be protected?

Reply:
How would this be protected?

* If aliased, it is protected along with other keyword aliases, I don't think anything special is needed for `@direction`.
* The _default base direction_ of a context is protected along with that context in the same way that _default language_ is protected.

* The _direction mapping_ of a term can be protected in the same way _language mapping_ is protected.

Reply:
> The default base direction of a context is protected along with that context in the same way that default language is protected.

If that's true already, this is what I was looking for. Great.